release.go

```diff
@@ -273,22 +297,15 @@ func buildReleaseCommits(ctx context.Context, ghClient *githubClient, commits []
 		}
 
 		if gen.UsePullRequestMetadata {
-			prNumber, ok := commit.PullRequestNumber()
-			if !ok {
-				continue
-			}
-			c.PullRequestNumber = prNumber
-
-			var err error
-			pr, ok := prs[prNumber]
-			if !ok {
-				pr, err = ghClient.getPullRequest(ctx, event.Owner, event.Repo, prNumber)
-			}
+			pr, err := getPullRequest(commit)
 			if err != nil {
 				return nil, err
 			}
-			c.PullRequestOwner = pr.GetUser().GetLogin()
-			c.ReleaseNote = extractReleaseNote(pr.GetTitle(), pr.GetBody(), gen.UseReleaseNoteBlock)
+			if pr != nil {
```

Comment:
ReleaseNote and PR Metadata is added only when a PR is obtained. In other words, even if a PR cannot be obtained, it will be included in the Release Commit list.